about.js

```diff
@@ -0,0 +1,20 @@
+const hamburger = document.querySelector('.hamburger');
+const navmenu = document.querySelector('.nav-menu');
+hamburger.addEventListener('click', () => {
+  if (document.body.style.overflow === 'hidden') {
+    document.body.style.overflow = 'auto';
+    hamburger.classList.remove('active');
+    navmenu.classList.remove('active');
+    /* eslint eqeqeq: "off", curly: "error" */
+  } else {
+    hamburger.classList.toggle('active');
+    navmenu.classList.toggle('active');
+    document.body.style.overflow = 'hidden';
+  }
+});
```

Comment:
- [ ] Great job on working on the menu but I think it would be great if you blured the background a little bit to let the focus just on the links because that's very important for the user experience , so kindly work on that 🙏🏻🙏🏻🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/183261390-c089901a-536a-4269-a7dc-1ee9472a52e4.png)